hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bucket/HoodieSimpleBucketIndex.java

```diff
@@ -44,7 +44,7 @@ public HoodieSimpleBucketIndex(HoodieWriteConfig config) {
     super(config);
   }
 
-  private Map<Integer, HoodieRecordLocation> loadPartitionBucketIdFileIdMapping(
+  public Map<Integer, HoodieRecordLocation> loadPartitionBucketIdFileIdMapping(
```

Comment:
Maybe keep it private.

Reply:
HoodieSimpleBucketIndex and RDDSimpleBucketPartitioner belong to different packages and can only use public.

Reply:
Could it be protected?